src/Symfony/Component/Console/Helper/Helper.php

```diff
@@ -45,6 +45,17 @@ public function getHelperSet()
      * @return int The length of the string
      */
     public static function strlen(?string $string)
+    {
+        return self::strwidth($string);
+    }
+
+    /**
+     * Returns the width of a string, using mb_strwidth if it is available.
+     * The width is how many characters positions the string will use.
+     *
+     * @internal in 5.2
```

Comment:
```suggestion
     * @internal since Symfony 5.2
```
As we do for other methods (ie `ChoiceFormField::containsOption`)

Reply:
there are several occurrences ;-)